tool/tsh/common/workload_identity.go

```diff
@@ -29,19 +29,24 @@ import (
 
 	"github.com/alecthomas/kingpin/v2"
 	"github.com/gravitational/trace"
-	"google.golang.org/protobuf/types/known/durationpb"
 
 	"github.com/gravitational/teleport"
-	machineidv1pb "github.com/gravitational/teleport/api/gen/proto/go/teleport/machineid/v1"
 	workloadidentityv1pb "github.com/gravitational/teleport/api/gen/proto/go/teleport/workloadidentity/v1"
 	"github.com/gravitational/teleport/api/types"
 	"github.com/gravitational/teleport/lib/client"
-	"github.com/gravitational/teleport/lib/cryptosuites"
 	"github.com/gravitational/teleport/lib/services"
 	"github.com/gravitational/teleport/lib/tbot/bot"
 	"github.com/gravitational/teleport/lib/tbot/workloadidentity"
 )
 
+const (
+	// Based on the default paths listed in
+	// https://github.com/spiffe/spiffe-helper/blob/v0.7.0/README.md
```

Comment:
```suggestion
// Based on the default paths listed in
// https://github.com/spiffe/spiffe-helper/blob/v0.7.0/README.md
const (
```

So that the godoc applies to the whole block instead of only to `svidPEMPath`.